andela/pippin-ah-backend
# [Feature: #162727574] Add Column for Cover Image Url to Article Model

#### What does this PR do?
This PR adds a cover image `URL` column to the `Article Model` and `controller`

#### Description of Task to be completed?
- Users should be able to add a cover image when creating an article

#### How should this be manually tested?
- run `db:migrate`
- the tab for `coverImageUrl` should be on your `Article` table
- you can now add a `coverImageUrl` param to `POST` request for articles

#### What are the relevant pivotal tracker stories?
[ #162727574](https://www.pivotaltracker.com/story/show/163863687)
